docs/installation/updating-with-npm.md

```diff
@@ -0,0 +1,34 @@
+# Updating GOV.UK Frontend using node package manager (NPM)
+
+You can update with node package manager (NPM) if you [originally installed
+GOV.UK Frontend with NPM](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#requirements).
+
+## Find out which version you're using
+
+You can find out which version of GOV.UK Frontend your project is using with:
```

Comment:
```suggestion
You can find out which version of GOV.UK Frontend your project is using by running:
```
Just a suggestion - it's a common docs pattern for commands that need to be run in the Terminal.

Reply:
Thanks Andrea - amended.